cmd/update.go

```diff
@@ -54,28 +55,24 @@ func newUpdateCmd() *updateCmd {
 			cfg := config.Get()
 
 			// Update single binary
+			binsToProcess := cfg.Bins
```

Comment:
nit: Can we move this line above the comment. It's a big misleading to understand that the comment is for the `if` statement and not this line :pray: